src/ovh-angular-mondial-relay/ovh-angular-mondial-relay.directive.js

```diff
@@ -72,11 +72,11 @@ angular.module("ovh-angular-mondial-relay")
         /**
          * Reformat the opening hours of the relays
          * @param {Object} opening Structure describing opening times {monday: [{start: "0800", end:"1900"}]}
-         * @returns {Array} List of open days [{day: "monday", opening: ["08:00-19:00"]}]
+         * @returns {Array} List of open days [{days: ["monday", "tuesday"], hours: ["08:00-19:00"]}]
          */
         var reformatOpening = function reformatOpening (opening) {
             var reformatTime = function (openingTime) {
-                return openingTime.substring(0, 2) + ":" + openingTime.substring(2, 4);
+                return openingTime.substring(0, 2) + "h" + (openingTime.substring(2, 4) === "00" ? "" : openingTime.substring(2, 4));
```

Comment:
Don't you have a time manipulation library instead ?

Reply:
Importing `moment.js` for one line of code is not the best to do imho...

Reply:
I think angular as a `date` filter that can be used for this purpose. If the component is not used in other countries than in France, it's alright though